apps/seg/Testing/CMakeLists.txt

```diff
@@ -29,5 +32,11 @@ add_test(NAME ${MODULE_NAME}_makeSEG
         --metaDataFileName ${CMAKE_SOURCE_DIR}/doc/seg-example.json
         --segImageFiles ${BASELINE}/liver_seg.nrrd
         --dicomImageFiles ${BASELINE}/01.dcm,${BASELINE}/02.dcm,${BASELINE}/03.dcm
-        --segDICOMFile ${TEMP_DIR}/liver.dcm
+        --segDICOMFile ${MODULE_TEMP_DIR}/liver.dcm
        )
+
+add_test(NAME seg_meta_roundtrip
+         COMMAND python ${CMAKE_SOURCE_DIR}/util/comparejson.py
```

Comment:
@fedorov The test is failing, because this python file does not exist. Its name is jsoncompare.py